website/src/components/flex/FlexCarousel.js

```diff
@@ -0,0 +1,7 @@
+const FlexCarousel = () => {
+  return (
+    <div>Carousel</div>
+  );
+}
+
+export default FlexCarousel;
```

Comment:
I wonder why we're getting no newlines on certain files. I figured prettier would have taken care of.

Reply:
When i set up my laptop again several weeks back I forgot to add editorconfig. Just installed it.